Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
+
+WORKDIR /opt/issue-sync
+
+ADD bin/issue-sync /opt/issue-sync/issue-sync
+
+CMD ["./issue-sync"]
```

Comment:
Please use the ENTRYPOINT command rather than CMD. When using CMD with no entrypoint, docker implicitly uses an entrypoint of /bin/sh. This means that if you `docker run quay.io/issue-sync <extra-args>` docker will run `/bin/sh -c '<extra-args>'`; to get this to work you would have to `docker run quay.io/issue-sync ./issue-sync <extra-args>`. Instead, if you add /opt/issue-sync` as the entrypoint, you can `docker run quay.io/issue-sync <extra-args>` and have them work as expected.